ament_clang_tidy/ament_clang_tidy/main.py

```diff
@@ -91,6 +94,11 @@ def main(argv=sys.argv[1:]):
         start_time = time.time()
 
     compilation_dbs = get_compilation_db_files(args.paths)
+    if args.packages_select:
+        if len(args.packages_select) == 1:
+            args.packages_select = args.packages_select[0].strip().split()
```

Comment:
Note that the point of these lines is that if someone has a newline-separated string of package names that they would like to use with this argument it will be handled nicely.  An example of a command that generates a newline-separated list of packages is `colcon list --names-only`.

Reply:
This is needed because if someone passes a newline-separated escaped string of package names it will be a one length list with the string.